index.html

```diff
@@ -12,40 +12,235 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>

```

Comment:
-[] Great job adding the README file to your project, but currently, the file is empty. Kindly follow the [Microverse Readme file ](https://github.com/microverseinc/readme-template)and it is set up for your project to be correctly documented.

Reply:
:red_circle: Your repository misses a descriptive README file. The title is not enough for the README.  Kindly follow this [Template](https://github.com/microverseinc/readme-template) and Please make sure you include all the necessary sections.